cli/src/lib/npm/npmLibDefs.js

```diff
@@ -470,65 +419,79 @@ export async function getInstalledNpmLibDefs(
   return installedLibDefs;
 }
 
+/**
+ * Retrieve single libdef.
+ */
+async function getItem(
```

Comment:
Could this be a little more specific, like `getSingleLibdef`?

Reply:
god point!